projects/client/RabbitMQ.Client/src/util/BlockingCell.cs

```diff
@@ -57,7 +57,6 @@ class BlockingCell<T>
     {
         private readonly ManualResetEventSlim _manualResetEventSlim = new ManualResetEventSlim(false);
         private T _value = default;
-        public EventHandler<T> ContinueUsingValue;
```

Comment:
Removed since this is unused. Build failed with the following (no idea why it started now):

```
src\util\BlockingCell.cs(60,32): error CS0649: Field 'BlockingCell<T>.ContinueUsingValue' is never assigned to, and will always have its default value null [C:\Users\lbakken\development\rabbitmq\rabbitmq-dotnet-client\projects\client\RabbitMQ.Client\Ra
bbitMQ.Client.csproj]
```

Reply:
I'm not seeing a failure related to this, but if it's not being used, no harm in removing it.